packages/asset/contracts/Catalyst.sol

```diff
@@ -41,6 +41,7 @@ contract Catalyst is
     OperatorFiltererUpgradeable
 {
     bytes32 public constant MINTER_ROLE = keccak256("MINTER");
+    bytes32 public constant BURNER_ROLE = keccak256("BURNER");
```

Comment:
For consistency with Asset.sol and OpenZeppelin guidance, let's make these:
bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
bytes32 public constant BURNER_ROLE = keccak256("BURNER_ROLE");